docs/fuzzing.md

```diff
@@ -13,12 +13,14 @@ vulnerabilities are not observable, and even the crash behaviour varies from cas
 to case. Furthermore, the goal to find a general approach for any authenticator
 rules out common solutions such as hardware emulation or binary instrumentation.
 
-## Corpus testing
+## How does it work
 
 Our idea is fuzzing by proxy. We take [OpenSK](https://github.com/google/OpenSK)
 (an open source implementation of a FIDO2 security key) as our fuzz target and 
 generate interesting input data guided by OpenSK's code coverage. At the moment,
-the fuzzing tool consists of running this input corpus on the device under test.
+we support two fuzzing modes: 
+1. Run this input corpus directly on the device under test.
+2. Fuzz the device under test using this input corpus as initial corpus.
```

Comment:
It would be the initial corpus, but then it would also be used for creating new data input right? (using the mutator?)

Reply:
Maybe you could say " Fuzz the device under test using mutations of this input corpus as initial corpus."

Reply:
I changed the wording to something similar.